style.css

```diff
@@ -28,3 +28,16 @@ a {
 	outline-width: 2px;
 	outline-style: solid;
 }
+
+/*
+ * Progresive enhancement to reduce widows and orphans
+ * https://github.com/WordPress/gutenberg/issues/55190
+ */
+h1, h2, h3, h4, h5, h6, blockquote, caption, figcaption {
+	text-wrap: balance; /* Safari fallback */
```

Comment:
`balance` can be an issue for CJK languages (and possibly others) ([see this core ticket](https://core.trac.wordpress.org/ticket/60892)). If it's just a fallback, I would suggest removing it and letting the fallback be default wrapping.

Reply:
@ryelle Thanks for letting me know. I followed the suggested approach on the GH issue and wasn't aware of the core ticket. I have removed `balance`.